src/Kros.AspNetCore/Authorization/ConnectionIdHelper.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Primitives;
+using System.Linq;
+
+namespace Kros.AspNetCore.Authorization
+{
+    /// <summary>
+    /// Helper class for working with connection id.
+    /// </summary>
+    public abstract class ConnectionIdHelper
+    {
+        /// <summary>
+        /// Connection ID.
+        /// </summary>
+        public const string ConnectionId = "Kros-Connection-Id";
```

Comment:
Toto tu nechceme. Celú knižnicu berieme ako všeobecnú, tak nechceme natvrdo do nej dávať veci špecifické pre KROS. Takže to treba prerobiť tak, aby sa toto dostalo do middleware-a zvonku. Už berie ako parameter nastavenia `GatewayJwtAuthorizationOptions`, tak treba do nich prirobiť novú vlastnosť podobne ako tam je `ForwardedHeaders` (napr. `public List<string> CacheHttpHeaders`) a pracovať s tým.

Reply:
spravil som tak